packages/flutter/test/widgets/semantics_tester.dart

```diff
@@ -119,11 +122,20 @@ class TestSemantics {
   /// they are created.
   final int id;
 
-  /// A bit field of [SemanticsFlags] that apply to this node.
-  final int flags;
+  /// The [SemanticsFlags] set on this node.
+  ///
+  /// There are two ways to specify this property: as an `int` that encodes the
+  /// flags as a bit field, or as a `List<SemanticsFlags>` that are _on_.
```

Comment:
Maybe include that the List is the preferred, more readable way?

Reply:
Done.